packages/openactive-broker-microservice/app.js

```diff
@@ -15,7 +15,7 @@ const WAIT_FOR_HARVEST = config.get('waitForHarvestCompletion');
 const ORDERS_FEED_REQUEST_HEADERS = config.get('ordersFeedRequestHeaders');
 const VERBOSE = config.get('verbose');
 
-process.env.NODE_TLS_REJECT_UNAUTHORIZED = 0;
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
```

Comment:
Env var values must be strings. This is also what the node.js code base expects (https://github.com/nodejs/node/search?q=NODE_TLS_REJECT_UNAUTHORIZED&unscoped_q=NODE_TLS_REJECT_UNAUTHORIZED)